movies/src/main/java/com/shotcutter/movies/TMDB/TMDBIntegrator.java

```diff
@@ -38,7 +39,7 @@
         this.movieRepository = movieRepository;
 
         pageSize = 20;
-        currentPage = ((int) movieRepository.count()) / pageSize;
+        currentPage = movieRepository.count().block().intValue() / pageSize;
```

Comment:
I'm not sure about the purpose of this. Usually we calculate pages during the request, we do not hold such info as singleton with 1 entry point. you don't even sync it

Reply:
I'll leave a refactoring todo